public/app/core/time_series2.ts

```diff
@@ -344,8 +344,8 @@ export default class TimeSeries {
 
   updateLegendValues(formater: ValueFormatter, decimals: DecimalCount, scaledDecimals: DecimalCount) {
     this.valueFormater = formater;
-    this.decimals = decimals;
-    this.scaledDecimals = scaledDecimals;
+    this.decimals = decimals || 0;
```

Comment:
nit: maybe consider using `??` instead of ||?
```suggestion
    this.decimals = decimals ?? 0;
```